docs/guides/src/main/server/fips.adoc

```diff
@@ -167,37 +167,36 @@ in the RHEL documentation for the details.
 The keycloak container image will be just automatically in fips mode when executed from the host in FIPS mode. 
 However, make sure that the Keycloak container also uses BCFIPS jars (instead of BC jars) and proper options when started.
 
-Regarding this, it is best to build your own container image as described in the <@links.server id="containers"/> and tweak it to use BCFIPS etc.
+Regarding this, it is best to build your own container image and tweak it to use BCFIPS etc.
 
-For example in the current directory, you can create sub-directory `files` and add:
+For example in the current directory, you can create sub-directory `keycloak` and add unpacked keycloak server, which you already prepared based on the steps above. So assumption is that it contains:
 
-* BC FIPS jar files as described above
-* Custom keystore file - named for example `keycloak-fips.keystore.bcfks`
-* Security file `kc.java.security` with added provider for SAML
+* BC FIPS jar files as described above in directory `providers`
+* Custom keystore file in `conf/server.keystore`
+* Security file `kc.java.security` with added provider for SAML in the `conf` directory
 
 Then create `Dockerfile` in the current directory similar to this:
 
 .Dockerfile:
 [source, dockerfile]
 ----
-FROM quay.io/keycloak/keycloak:latest as builder
-
-ADD files /tmp/files/
+FROM registry.access.redhat.com/ubi9/ubi:latest as builder
 
+ADD keycloak /opt/keycloak/
 WORKDIR /opt/keycloak
-RUN cp /tmp/files/*.jar /opt/keycloak/providers/
-RUN cp /tmp/files/keycloak-fips.keystore.* /opt/keycloak/conf/server.keystore
-RUN cp /tmp/files/kc.java.security /opt/keycloak/conf/
 
-RUN /opt/keycloak/bin/kc.sh build --features=fips --fips-mode=strict
+RUN dnf install -y java-17-openjdk-devel
+RUN /opt/keycloak/bin/kc.sh build  --features=fips --fips-mode=strict
```

Comment:
To keep the examples consistent with how you usually set options and how we have it documented in our guides, we should favor using environment variables when doing container/k8s.